From 50d13805262202627e0a8494508202d21a86c18b Mon Sep 17 00:00:00 2001
From: Nikolai Kondrashov <Nikolai.Kondrashov@redhat.com>
Date: Mon, 27 Oct 2014 16:24:09 +0200
Subject: [PATCH 2/2] man: Add -P option to radtest synopsis

Add "-P" option to radtest's manpage SYNOPSIS to make it match the
OPTIONS section.
---
 man/man1/radtest.1 | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/man/man1/radtest.1 b/man/man1/radtest.1
index 587870c..b318477 100644
--- a/man/man1/radtest.1
+++ b/man/man1/radtest.1
@@ -5,6 +5,8 @@ radtest - send packets to a RADIUS server, show reply
 .B radtest
 .RB [ \-d
 .IR raddb_directory ]
+.RB [ \-P
+.IR tcp/udp ]
 .RB [ \-t
 .IR pap/chap/mschap/eap-md5 ]
 .RB [ \-x
-- 
2.1.1

